### DISCUSSION TITLE
GraphQL Schema Registry

### DISCUSSION THREAD

**Discussion by sbilello** [Netflix/dgs-framework#228] [Q&A] [answered]
Hello!

Which GraphQL Schema Registry is recommended to use in the case of the GraphQL Federated approach?

I found this one [pipedrive-graphql-schema-registry](https://github.com/pipedrive/graphql-schema-registry). Is there any recommended one?

Thanks

**Comment by paulbakker** [comment, marked as answer]
At Netflix we have our own implementation. The only other implementation I'm familiar with (and can recommend) is Apollo's offering.

**Reply by sabarikrishnavk** [reply]
@paulbakker any plans to open source the schema registry ? is it build over spring boot?

**Comment by aisensiy** [comment]
`pipedrive-graphql-schema-registry` is just text diff. Apollo's offering do a better job.

**Reply by mcacek** [reply]
The guild recently released a new project that can be self-hosted or used as a SaaS offering - https://graphql-hive.com/ This looks promising as a potential Apollo Studio alternative.

**Comment by kamilkisiela** [comment]
@sbilello I'm from [The Guild](https://the-guild.dev), so of course I'm biased, but I think [GraphQL Hive](https://graphql-hive.com/) is a nice Schema Registry that is fully open source and can be self-hosted. The goal was to have a GraphQL Schema Registry that everyone could contribute to and that would be build in public.
